<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GmailConfigDialog</class>
 <widget class="QWidget" name="GmailConfigDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>460</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="generalTab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="accountBox">
         <item>
          <widget class="QGroupBox" name="currentAccountBox">
           <property name="title">
            <string>Account</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
            <item>
             <widget class="QLabel" name="label_1">
              <property name="text">
               <string>Authenticated as:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="usernameLabel">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KPushButton" name="changeAuthButton">
              <property name="text">
               <string>Change...</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="KPushButton" name="authenticateButton">
         <property name="text">
          <string>Authenticate</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="mailCheckBox">
         <property name="title">
          <string>Mail Checking Options</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QCheckBox" name="disconnectedModeEnabled">
            <property name="text">
             <string>&amp;Download all messages for offline use</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="enableMailCheckBox">
            <property name="text">
             <string>Enable &amp;interval mail checking</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Check mail interval:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KIntNumInput" name="checkInterval">
              <property name="value">
               <number>15</number>
              </property>
              <property name="minimum">
               <number>5</number>
              </property>
              <property name="suffix">
               <string/>
              </property>
              <property name="prefix">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="advancedTab">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>IMAP Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="subscriptionEnabled">
            <property name="text">
             <string>&amp;Synchronize only selected folders</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KPushButton" name="subscriptionButton">
            <property name="text">
             <string>Select folders to synchronize...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="QCheckBox" name="autoExpungeCheck">
            <property name="text">
             <string>Automatical&amp;ly compact folders (expunge deleted messages)</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="identityBox">
         <property name="title">
          <string>Identity Settings</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="useDefaultIdentityCheck">
            <property name="text">
             <string>Use &amp;default identity</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="identityLayout">
            <item>
             <widget class="QLabel" name="identityLabel">
              <property name="text">
               <string>Identity:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KIntNumInput</class>
   <extends>QWidget</extends>
   <header>knuminput.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
